### PR TITLE
Forms Automated Test - LPS-133117 | master 

### PR DESCRIPTION
**Pull request by evanilson-santana**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/macros/Form.macro [hunk 1]
```diff
@@ -681,6 +681,18 @@ definition {
 		AssertElementPresent(locator1 = "Form#ALT_PAGINATION_SUCCESS_PAGE");
 	}
 
+	macro viewFieldReference {
+		key_fieldLabel = "${fieldLabel}";
+
+		FormViewBuilder.clickFieldLabel(fieldLabel = "${key_fieldLabel}");
+
```
Comment (alinedoleron) [inline portal-web/test/functional/com/liferay/portalweb/macros/Form.macro:688]: Hi @evanilson-santana , I just noticed that FormViewBuilder.clickFieldLabel macro belongs to AppBuilder, not to Forms. Maybe you could try the Click function. Please, check if this example I found in some testcases is applicable to this case.

```
Click(key_name = "Numeric", locator1 = "FormFields#FIELD_TARGET_LABEL");
```
Thanks! =)

**Review by alinedoleron — CHANGES_REQUESTED** [review]
Hi @evanilson-santana, I've added a comment regarding the usage of App Builder macros instead of Forms product macros. This must be avoided, since App Builder folder will be deleted soon. Please, take a look on it.

Good job! =)

**Comment by alinedoleron** [comment]
In addition, it seems there're some conflicts here. Please, rebase it. Thank you!

**Comment by evanilson-santana** [comment]
Hi @alinedoleron All test steps are running correctly, I updated the macro for the specific test.

**Comment by alinedoleron** [comment]
Good job, @evanilson-santana! Please, could you generate a new index.html to put in the related [Jira ticket](https://issues.liferay.com/browse/LPS-133117)? Just for keep the tracking of the executions =) After this, I'll forward the PR. Thank you very much!

**Pull request closed without merging**